package.json

```diff
@@ -56,6 +56,7 @@
     "google-map-react": "^0.20.0",
     "insert-css": "^1.0.0",
     "jest-cli": "^16.0.0",
+    "jsdoc-parse": "^1.2.7",
```

Comment:
Documentation on v2 is very scarce, and I don't get the intent is for this next version as it mentions right away the use of jsdoc2md, but we don't want the markdown transformation https://github.com/jsdoc2md/jsdoc-parse